src/styles/services-view.less

```diff
@@ -223,16 +230,15 @@ services-view,
           margin-left: -10px;
           opacity: 0;
           position: absolute;
-          transition: opacity 100ms linear;
         }
         .services-sub-category-tab-image img {
           opacity: 1;
         }
       }
 
-      &.active.items-shown {
-        .services-sub-category-tab:after,
-        .services-items {
+      &.active {
+        .services-sub-category-tab:after {
```

Comment:
These should be part of the rule at line 220, no?  Line 231 sets the opacity to 0, but this rule sets it back to 1.